### PR TITLE
Grid component

### PR DESCRIPTION
**Pull request by EdithLansens**
New visualization available for DataMiner Dashboards and Low-Code Apps: the grid component.

Currently only available if ReportsAndDashboardsDynamicVisuals soft-launch option is enabled.

RNs: 34761, 34781, 34948, 37269, and 37699.

### CODE REVIEW

**Comment by EdithLansens** [comment]
Hi @GillesBara, @simonmestdagh2, @WaHaeck, and @SebastiaanSL, could you review this pull request to ensure the accurate inclusion of the grid component in the user guide? More information is set to follow about the Template Editor in an upcoming pull request.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+> [!IMPORTANT]
+> At present, this component is only available in preview, if the [ReportsAndDashboardsDynamicVisuals](xref:Overview_of_Soft_Launch_Options#reportsanddashboardsdynamicvisuals) soft-launch option is enabled. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
+
+Available from DataMiner 10.4.1/10.5.0<!--RN 34761-->. This component allows you to visualize data as a grid.
+
+To configure the component:
+
+1. Apply a data feed. See [Applying a data feed](xref:Apply_Data_Feed).
+
+1. Optionally, hover the mouse over the component, click the filter icon, and then add a filter feed from the *queries* section of the data pane. You can repeat this several times in order to filter on several queries<!--RN 34761-->.
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md:16]: The only filter that currently makes sense to the grid is the columns from the query filter component, which will be available under Feeds > Query filter >Query Columns

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+uid: DashboardGrid
+---
+
+# Grid
+
+> [!IMPORTANT]
+> At present, this component is only available in preview, if the [ReportsAndDashboardsDynamicVisuals](xref:Overview_of_Soft_Launch_Options#reportsanddashboardsdynamicvisuals) soft-launch option is enabled. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md:8]: The component is available without softlaunch from 10.4.1. Between 10.2.12 and 10.4.1 it's available with the softlaunch. This note makes me think I can only use the component with the flag enabled from 10.4.1 onwards.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+       - ![Scaled to fit (scaling)](~/user-guide/images/Scaling.png) : The scaling of the cells in the columns and/or rows is set to a fixed size. This is the default setting.
+
+     > [!NOTE]
+     >
+     > - The number of items that can be displayed in a grid component is limited to 1000<!--RN 37699-->.
+     > - If the number of items to be displayed exceeds the number of cells displayed in the component, navigation buttons are available to navigate through the data<!--RN 34761-->.
+     > - When the scaling of the cells is set to a fixed size and there are too many columns and/or rows to show them at once in the component, in read mode, it is possible to scroll through them with a scrollbar that becomes visible when you hover over the component<!--RN 37699-->.
+
+   - *Item templates > Alarms*: Allows you to freely customize the appearance of the grid component using templates<!--RN 34761-->.
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md:62]: This "Alarms" is the name of the query, not a fixed part of the setting name

**Review by GillesBara — CHANGES_REQUESTED** [review]

**Comment by EdithLansens** [comment]
Hi @GillesBara, thanks for the feedback! I've made the associated changes in [commit 7704d11](https://github.com/SkylineCommunications/dataminer-docs/pull/2291/commits/7704d11b1cd1cfeaa62ad4d8684f1072c18700e5).

**Review by GillesBara — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**